tests/config/ncsa/ForwarderCfg.yaml

```diff
@@ -0,0 +1,57 @@
+# name of forwarder
+NAME: f99
+
+# logging directory
+LOGGING_DIR: ./
+
+# working directory where cfitio and header files are fetched/assembled
+WORK_DIR: ./
+
+# daq partition name
+PARTITION: ncsa-comcam
+FOLDER: .templates
+
+# LCA-13501 segment order
+DAQ_MAPPING: [ 10, 11, 12, 13, 14, 15, 16, 17, 07, 06, 05, 04, 03, 02, 01, 00 ]
+HDR_MAPPING: [ 10, 11, 12, 13, 14, 15, 16, 17, 07, 06, 05, 04, 03, 02, 01, 00 ]
+
+# daq location patterns in the hardware. These values are pre-defined here so
+# that forwarder can throw exception when someone gives wrong "raft/ccd"
+# location. Currently daq hardware segfaults when wrong argument is given to
+# constructor
+ats: [ 00/0 ]
+ncsa-comcam: [ 22/0, 22/1, 22/2 ]
+
+# rabbitMQ broker address
+BASE_BROKER_ADDR: 141.142.238.15:5672/%2ftest_cc
+
+# @deprecated_soon
+# forwarder queue to listen to RabbitMQ messages. This is static queue, meaning
+# someone has to create this queue in advance for forwarder to work. In the
+# future, forwarder is gonna create queues on the fly when it starts up.
+CONSUME_QUEUE: f99_consume
+
+# rabbitMQ archivecontroller queue
+ARCHIVE_QUEUE: archive_ctrl_consume
+
+# telemetry queue for publishing telemetry messages from forwarder
+TELEMETRY_QUEUE: telemetry_queue
+
+# redis database configuration for handling heartbeat and storing information
+REDIS_HOST: localhost
+REDIS_PORT: 6379
+REDIS_DB: 1
```

Comment:
should this be 14?

Reply:
This is for local testing purpose. So, I did not want to use the production database number.